### DISCUSSION TITLE
Why 5 events to get new list of validator order??

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#121] [General]
Why does it need 5 events vefore updating the order of the validator list? As i understood the validator list is displayed randomly right, each new event should display a new random order ? or ? 

See the video : I create 5 events and it changes : 

https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/307424b6-05ab-4e44-a122-21641be6e27c

**Comment by TJ-BitCanna** [comment]
List is refreshed every time the page is refreshed.

If you stay on the page, the list will be the same. 
But 2 different users will always see a different order of the list.
